### DISCUSSION TITLE
Underworld feature

### DISCUSSION THREAD

**Discussion by Liizzeey** [the-marenga/sf-scrapbook-helper#96] [Q&A] [answered]
What is the use of the underworld feature? One of my characters has a lure levelof 426 and the tool suggests level 10 characters and the highest one is level 153

**Comment by the-marenga** [comment, marked as answer]
The underworld tab is only showing you enemies, that do not have any armor/weapons, so that you can lure the highest level player possible. If you are just seeing low level players, then that is either a bug, or just means that there are no "naked" high level players on the server
